android/src/main/java/com/rngocoder/BroadcastView.java

```diff
@@ -93,9 +98,16 @@ public void onHostResume() {
             cameraView.startPreview();
         }
 
-    }
-
+        if (goCoder != null && cameraView != null) {
+            if (mAutoFocusDetector == null)
+                mAutoFocusDetector = new GestureDetectorCompat(localContext, new AutoFocusListener(localContext, cameraView));
```

Comment:
This line file to compile, the "AutoFocusListener" class can't be find in this PR.  I assume you're using the same from the GoCoderAndroid Sdk example, please include it.